modules/ingester/instance_search.go

```diff
@@ -7,6 +7,10 @@ import (
 	"strings"
 	"sync"
 
+	ot_log "github.com/opentracing/opentracing-go/log"
+	"github.com/pkg/errors"
```

Comment:
aren't Is and As in the standard errors package? i wonder why we import this

Reply:
We are using both packages `errors` and `github.com/pkg/errors` in the code base. And we are also using both wrapping techniques `fmt.Errorf("%w")` from the standard library and `errors.Wrap()` from `github.com/pkg/errors`. It isn't clear to me which one we prefer. For this PR I made the decision to use `github.com/pkg/errors` where both packages can be used.

Reply:
i'm surprised `Wrap` isn't in the standard library. I think this PR is big enough as is, but we should consider standardizing in the future.

Reply:
> I think this PR is big enough as is, but we should consider standardizing in the future

I created this issue to replace github.com/pkg/errors with errors #2946